package.json

```diff
@@ -20,15 +21,22 @@
     "axios": ">=0.27"
   },
   "devDependencies": {
+    "@babel/core": "^7.20.5",
+    "@babel/preset-env": "^7.20.2",
+    "@babel/preset-typescript": "^7.18.6",
+    "@jest/globals": "^29.3.1",
```

Comment:
During installation got such an error:
```
error @jest/globals@29.3.1: The engine "node" is incompatible with this module. Expected version "^14.15.0 || ^16.10.0 || >=18.0.0". Got "16.4.0"
```

Let's specify .node-version or .tool-versions (for asdf) for our repo
- I suggest using LTS right now it's 18.12.1

Reply:
Done.